build/utils.py

```diff
@@ -221,8 +122,11 @@ def process_accelerate_launch_args(job_config_dict):
         )
 
     # Add training_script
-    script = os.environ.get("LAUNCH_TRAINING_SCRIPT", "/app/launch_training.py")
-    accelerate_launch_args.append(script)
+    script = os.environ.get("TRAINING_SCRIPT")
+    if script:
+        accelerate_launch_args.append(script)
+    else:
+        accelerate_launch_args.extend(["--module", "tuning.sft_trainer"])
```

Comment:
Note the new way of calling sft_trainer. Instead of calling a python script, calling the python module 
accelerate launch docs note:
> Change each process to interpret the launch script as a Python module, executing with the same behavior as ‘python -m

Reply:
Not really, this is an edge case for extensability but is not likely to come up